src/Shouldly/Configuration/VisualStudioDiffTool.cs

```diff
@@ -50,6 +50,7 @@ private static string GetPath()
                 return null;
             }
 
+#if !NETSTANDARD2_0
             if (process != null)
```

Comment:
Not sure what to do with `ProcessModule` missing `FileName` property in `netstandard20`